ci-operator/step-registry/fips-check/node-scan/fips-check-node-scan-commands.sh

```diff
@@ -0,0 +1,109 @@
+    <failure message="">Fips node scan check failed</failure>
+  </testcase>
+</testsuite>
+EOF
+    fi
+}
+
+function set_proxy () {
+    if test -s "${SHARED_DIR}/proxy-conf.sh" ; then
+        echo "setting the proxy"
+        echo "source ${SHARED_DIR}/proxy-conf.sh"
+        source "${SHARED_DIR}/proxy-conf.sh"
+    else
+        echo "no proxy setting."
+    fi
+}
+
+function run_command() {
+    local CMD="$1"
+    echo "Running Command: ${CMD}"
+    eval "${CMD}"
+}
+
+set_proxy
+run_command "oc whoami"
+run_command "oc version -o yaml"
+
+# skip for ARM64
+mapfile -t node_archs< <(oc get nodes -o jsonpath --template '{range .items[*]}{.status.nodeInfo.architecture}{"\n"}{end}' | sort -R | uniq -d)
+echo "The architectures included in the current cluster nodes are ${node_archs[*]}"
+NON_SUPPORTED_ARCHES=(arm64)
+for arch in "${node_archs[@]}"; do
+    if [[ "${NON_SUPPORTED_ARCHES[*]}" =~ $arch ]]; then
+        echo "Skip this test since it doesn't support $arch."
+        exit 0
+    fi
+done
+
+# get a master node
+master_node_0=`oc get node -l node-role.kubernetes.io/master= --no-headers | grep -Ev "NotReady|SchedulingDisabled"| awk '{print $1}' | awk 'NR==1{print}'`
+if [[ -z $master_node_0 ]]; then
+    echo "Error master node0 name is null!\n."
+    exit 1
+fi
+# create a ns
+project="node-scan-$RANDOM"
+run_command "oc new-project $project --skip-config-write"
+if [ $? == 0 ]; then
+    echo "create $project project successfully" 
+else
+    echo "fail to create $project project."
+fi
+# check whether it is disconnected cluster
+oc label namespace "$project" security.openshift.io/scc.podSecurityLabelSync=false pod-security.kubernetes.io/enforce=privileged pod-security.kubernetes.io/audit=privileged pod-security.kubernetes.io/warn=privileged --overwrite=true
+cluster_http_proxy=`oc get proxy cluster -o=jsonpath='{.spec.httpProxy}'`
+cluster_https_proxy=`oc get proxy cluster -o=jsonpath='{.spec.httpProxy}'`
+attempt=0
+while true; do
+    out=$(oc --request-timeout=60s -n "$project" debug node/"$master_node_0" -- chroot /host bash -c "export http_proxy=$cluster_http_proxy; curl -sSI ifconfig.me --connect-timeout 5" 2> /dev/null || true)
+    if [[ $out == *"Via: 1.1"* ]]; then
+        echo "This is not a disconnected cluser"
+        break
+    fi
+    attempt=$(( attempt + 1 ))
+    if [[ $attempt -gt 3 ]]; then
+        echo "This is a disconnected cluser, skip testing"
+        oc delete ns "$project"
+        exit 0
+    fi
+    sleep 5
+done
+
+# run node scan and check the result
+report="/tmp/fips-check-node-scan.log"
+oc --request-timeout=300s -n "$project" debug node/"$master_node_0" -- chroot /host bash -c "export http_proxy=$cluster_http_proxy; export https_proxy=$cluster_https_proxy; podman run --authfile /var/lib/kubelet/config.json --privileged -i -v /:/myroot registry.ci.openshift.org/ci/check-payload:latest scan node --root  /myroot 2>&1> $report"
```

Comment:
This part:

```sh
2>&1> $report
```
won't work as expected. Here's a repro:

```console
[kir@kir-rhat uapi]$ cat a.sh
echo foo
echo bar >&2
[kir@kir-rhat uapi]$ ./a.sh 2>&1> out
bar
[kir@kir-rhat uapi]$ cat out
foo
```

The correct way to write this would be 

```sh
> report 2>&1
```

or, if using bash, you can just do
```sh
&> $report
```